packages/terra-embedded-content-consumer/tests/wdio/embedded-content-consumer-spec.js

```diff
@@ -39,12 +23,32 @@ describe('Embedded Content Consumer', () => {
     before(() => {
       browser.url('#/raw/tests/terra-embedded-content-consumer/embedded-content-consumer/consumers/custom-events-consumer');
       browser.waitForExist('#CustomEvents');
+      browser.pause(1000);
+    });
+
+    Terra.should.matchScreenshot('EventA', { viewports });
+
+    it('successfully replyed with EventA message', () => {
+      const myFrame = browser.element('iframe[src="#/raw/provider/terra-embedded-content-consumer/embedded-content-consumer/providers/custom-events-provider"]').value;
+      browser.frame(myFrame);
+
+      expect(browser.getText('#embedded-content-consumer-reply')).to.have.string('eventA');
+      browser.frameParent();
+    });
 
+    it('waits for Provider to trigger eventB', () => {
       // Waiting for events to execute.
-      browser.pause(5000);
+      browser.pause(10000);
```

Comment:
👀

Reply:
I increased the timeout in the example so people viewing the example on the doc site can see the events take place.[Previously after one second](https://github.com/cerner/terra-framework/pull/545/files#diff-dd84921fe32bf00761aacc4a323c3325L15), the border color would change. Now after [10 seconds](https://github.com/cerner/terra-framework/pull/545/files#diff-c16ef7a44ef9df26577dc8a012f9f546R17) the border color update will occur.